server/src/test/java/io/crate/protocols/postgres/PostgresWireProtocolTest.java

```diff
@@ -726,6 +726,115 @@ public void testHandleCancelRequestBody() {
         assertThat(channel.isOpen()).isFalse();
     }
 
+    @Test
+    public void test_throw_error_on_if_startup_message_is_to_short() {
+        Sessions sqlOperations = mock(Sessions.class);
+        Session session = mock(Session.class);
+        when(session.sessionSettings()).thenReturn(CoordinatorSessionSettings.systemDefaults());
+        when(sqlOperations.newSession(
+            any(ConnectionProperties.class),
+            any(String.class),
+            any(Role.class))
+        ).thenReturn(session);
```

Comment:
```suggestion
```

There's a `sqlOperations` created in the setup and it looks like the tests all work using that.

Reply:
Good catch, thanks. I'll also remove the not-needed mocks of other tests here.